extensions/doc/init.lua

```diff
@@ -120,8 +120,8 @@ docMT = {
         return fixLinks(result)
     end,
     __pairs = function(self)
-        local path, pos = rawget(self, "__path"), rawget(self, "__pos")
-        local source = {}
+        local _, pos = rawget(self, "__path"), rawget(self, "__pos")
```

Comment:
If we're not using `path` here then rather than make it `_` we could probably remove it entirely and just have `local pos = rawget(self, "__pos")` right?